### ISSUE TITLE
Batch normalization in VGG16

### ISSUE THREAD

**Issue by TiJoy** [bryanyzhu/two-stream-pytorch#34]
Hi, I just want to know that if batch_norm will influence the performance in rgb_vgg16, because it looks like that you didn't use batch_norm in rgb_vgg16.
`def make_layers(cfg, batch_norm=False):`
`model = VGG(make_layers(cfg['D']), **kwargs)`
I saw there is a rgb_vgg16_bn function but it has no available pretrained module, so how about its performance?

**Comment by bryanyzhu** [comment]
I didn't experiment with VGG16BN it actually. I train the VGG16 model just to compare its performance with the original paper, so that I can know I'm on the right track. 

Since VGG16 model is pretty huge, it's better to use some other models like ResNet. 

But the VGG16BN model should work. If you are really interested in its performance, you can train it yourself.

**Comment by bryanyzhu** [comment]
Close the issue for now.
